packages/main/src/Avatar.ts

```diff
@@ -350,12 +351,24 @@ class Avatar extends UI5Element implements ITabbable {
 		this._checkInitials();
 	}
 
+	onEnterDOM() {
+		ResizeHandler.register(this, this._resizeHandler.bind(this));
```

Comment:
In onEnterDOM 
the bind  `this._resizeHandler.bind(this)` returns a new function

Later in onExitDOM
`this._resizeHandler.bind(this)` returns a new function again and it does not de-register the previously attached function onEnterDOM

We need to register/deregister the same function, otherwise it's the same as if we haven't called deregister at all - it does not prevent memory leaks